blosc/tests/test.rs

```diff
@@ -63,7 +68,11 @@ test_suite! {
         }).collect::<Vec<_>>();
 
 
-        let encoded = settings.val.compress(&sample[..]);
+        let encoder = settings.val;
+        if encoder.is_err() {
+            return
```

Comment:
Why return?  That would pass the test.  Better to panic in this case.

Reply:
Not all versions of blosc ships with all decoders. Better to have a fail-open and ensure the CI uses all encoders.